### PR TITLE
583 full test suite

### PR DESCRIPTION
**Pull request by rozhkovdmitrii**

### CODE REVIEW

File: .buildkite/steps/push-readme.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+docker run --rm -t -v $(pwd)/:/full_test_suite -e DOCKER_USER='rozhkovdmitrii' -e DOCKER_PASS='7d5bcd02' \
```
Comment (gigimon) [inline .buildkite/steps/push-readme.sh]: password open

**Review by gigimon — DISMISSED** [review]
get password for docker hub from env

File: .buildkite/steps/push-readme.sh (hunk 1 above)
Comment (rozhkovdmitrii) [reply .buildkite/steps/push-readme.sh]: OMG

File: .buildkite/steps/push-readme.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    headers["Accept"] = "application/json"
+    headers["Authorization"] = "Bearer " + token
+    headers["Content-Type"] = "application/json"
+    logger.debug("Open DOCKERHUB.md")
+    with open("DOCKERHUB.md") as readme_file:
+        readme_data = readme_file.read()
+        json_data = json.dumps({"full_description": readme_data})
+        url = "https://hub.docker.com/v2/repositories/neonlabsorg/full_test_suite/"
+        logger.debug("Request (patch) to update full_description at " + url)
+        response = requests.patch(url, data=json_data, headers=headers)
```
Comment (gigimon) [inline .buildkite/steps/push-readme.py]: requests accept parameter json and automatically make dumps

**Review by gigimon — DISMISSED** [review]

File: .buildkite/steps/push-readme.py (hunk 2 above)
Comment (rozhkovdmitrii) [reply .buildkite/steps/push-readme.py]: solved

**Review by gigimon — APPROVED** [review]

**Pull request merged**